.github/workflows/ci.yml

```diff
@@ -83,7 +85,29 @@ jobs:
                   toolchain: stable
                   override: true
                   components: clippy
+
+            - name: Cache ArrayFire
+              uses: actions/cache@v1
+              id: arrayfire
+              with:
+                  path: afbin
+                  key: ${{ runner.os }}-af-${{ env.AF_VER }}
+
+            - name: Download ArrayFire
```

Comment:
I wonder if we can re-use this cache from both the jobs rather than doing it twice in the same workflow. Can you please look into it.

Reply:
I _think_ that the cache is shared between jobs. https://github.com/actions/cache/issues/549 seems to confirm this. So -- as I understand it -- we should re-use the same cache entry for both jobs if we use the same cache key.